src/servers/trigger.ts

```diff
@@ -178,16 +180,16 @@ export const onchainBlockEventHandler = async (height: number) => {
   logger.info(`finish block ${height} handler with ${txNumber} transactions`)
 }
 
-export const invoiceUpdateEventHandler = async (invoice: GetInvoiceResult) => {
+export const invoiceUpdateEventHandler = async (
+  invoice: SubscribeToInvoiceInvoiceUpdatedEvent,
+): Promise<boolean | ApplicationError> => {
```

Comment:
I think at some point (not necessarily with this PR) we should have `Success` instead of `boolean`. if if this is an alias to a boolean, this would give a better insights into what is being returned.